README.md

```diff
@@ -284,7 +284,7 @@ bash -c "$(curl https://raw.githubusercontent.com/commaai/openpilot/master/tools
 ```
 3) Press 1 to engage openpilot
 
-See the full [README](tools/sim/README.md)
+For further information see the full [README](tools/sim/README.md), as well as the [CARLA](https://github.com/commaai/openpilot/wiki/CARLA) page on the openpilot wiki
```

Comment:
let's revert this since it's linked in the sim README

Reply:
Understandable!

Not certain if there's an easier method, but I went ahead and pushed a new commit which reverts all changes to the main README.